webapp/src/js/controllers/reports.js

```diff
@@ -338,27 +340,27 @@ angular
       Modal({
         templateUrl: 'templates/modals/edit_report.html',
         controller: 'EditReportCtrl',
-        model: { report: ctrl.selected[0].doc },
+        model: { report: ctrl.selectedReports[0].doc },
       });
     });
 
     $scope.$on('VerifyReport', function(e, valid) {
-      if (ctrl.selected[0].doc.form) {
+      if (ctrl.selectedReports[0].doc.form) {
```

Comment:
Looks like ~5 references to `ctrl.selectedReports[0].doc` in this function. Maybe alias it for more concise code?

Reply:
That might be a bit complicated as `ctrl.selectedReports` is constantly being updated, so I could make a selector for it but that might be overkill as it's not really a globally relevant value.